client/src/app/videos/+video-watch/comment/video-comment.component.ts

```diff
@@ -78,16 +79,16 @@ export class VideoCommentComponent implements OnInit, OnChanges {
   }
 
   isRemovableByUser () {
-    return this.comment.account && this.isUserLoggedIn() &&
+    return this.isUserLoggedIn() &&
```

Comment:
why did you remove `this.comment.account`?

Reply:
Ah, that's an artifact from the messy conflict resolution I went through. Sorry about that.